tests/Congress.test.ts

```diff
@@ -1697,3 +1726,174 @@ describe("Test for the removing of validators", () => {
+        if (!SodiumHelper.isAssigned()) SodiumHelper.assign(new BOASodium());
+        await SodiumHelper.init();
+    });
+
+    before("Start a fake Agora", () => {
+        return new Promise<void>((resolve, reject) => {
+            agora_server = new TestAgora(agora_addr.port, sample_data, resolve);
+        });
+    });
+
+    before("Start a fake TestCoinGeckoServer", () => {
+        return new Promise<void>(async (resolve, reject) => {
+            gecko_server = new TestGeckoServer("7876", market_cap_sample_data, market_cap_history_sample_data, resolve);
+            gecko_market = new CoinGeckoMarket(gecko_server);
+        });
+    });
+
+    before("Start a fake TestCoinGecko", () => {
+        coinMarketService = new CoinMarketService(gecko_market);
+    });
+
+    before("Start a fake votera Server and Service", () => {
+        return new Promise<void>(async (resolve, reject) => {
+            votera_server = new TestVoteraServer(1337, votera_addr, resolve);
+            votera_service = new VoteraService(votera_addr);
+        });
+    });
+    before("Create TestStoa", () => {
+        testDBConfig = MockDBConfig();
+        stoa_server = new TestStoa(
+            testDBConfig,
+            agora_addr,
+            parseInt(stoa_addr.port, 10),
+            votera_service,
+            coinMarketService
+        );
+        return stoa_server.createStorage();
+    });
+
+    before("Start TestStoa", async () => {
+        await stoa_server.start();
+        await stoa_server.voteraService?.stop();
+        return;
+    });
+
+    after("Stop Stoa and Agora server instances", async () => {
+        await stoa_server.ledger_storage.dropTestDB(testDBConfig.database);
+        await stoa_server.stop();
+        await votera_server.stop();
+        await gecko_server.stop();
+        await agora_server.stop();
+    });
+
+    it("Test of writing proposal transaction", async () => {
+        const url = URI(stoa_private_addr).directory("block_externalized").toString();
+        await client.post(url, { block: sample_data6 });
+        await delay(200);
+        await client.post(url, { block: sample_data3 });
+        await delay(200);
+
+
+        //  Verifies that all sent blocks are wrote
+        const uri = URI(stoa_addr).directory("/block_height");
+        const response = await client.get(uri.toString());
+
+        assert.strictEqual(response.status, 200);
+        assert.strictEqual(response.data, "3");
+    });
+
+    it("Start votera service for syncing proposal's meta information", async () => {
+        await stoa_server.voteraService?.start(stoa_server, 2);
+        await delay(200);
+
+        await stoa_server.voteraService?.stop();
+    });
+
+    it("Test of writing ballot transaction", async () => {
+        const url = URI(stoa_private_addr).directory("block_externalized").toString();
+        await client.post(url, { block: sample_data7 });
```

Comment:
What is the 'sample_data7' ?

Reply:
@MichaelKim20 this for writing static ballot transaction into the test DB